pkgs/tools/networking/zap/default.nix

```diff
@@ -1,30 +1,25 @@
-{ lib, stdenv, fetchFromGitHub, jdk8, ant, runtimeShell }:
+{ lib, stdenv, jdk8, bash, runtimeShell }:
 
-let jdk = jdk8; in
-stdenv.mkDerivation rec {
+let jdk = jdk8;
+in stdenv.mkDerivation rec {
   pname = "zap";
-  version = "2.7.0";
-  src = fetchFromGitHub {
-    owner = "zaproxy";
-    repo = "zaproxy";
-    rev =version;
-    sha256 = "1bz4pgq66v6kxmgj99llacm1d85vj8z78jlgc2z9hv0ha5i57y32";
+  version = "2.10.0";
+  src = fetchTarball {
+    url = "https://github.com/zaproxy/zaproxy/releases/download/v${version}/ZAP_${version}_Linux.tar.gz";
```

Comment:
Why are we not building it?

Reply:
Had no success with building from github (and didn't see others (Arch, Gentoo)) building from source so, decided to use tarball as other package managers do.